src/components/Board/Board.container.js

```diff
@@ -1189,24 +1189,24 @@ export class BoardContainer extends Component {
     this.props.history.replace(`/board/${boardId}`);
   }
 
-  onRequestPreviousBoard() {
+  onRequestPreviousBoard = () => {
```

Comment:
These changes are not related. This increases the size of the PR make it difficult to review and adding complexity to merge with existing branches